packages/prisma/schema.prisma

```diff
@@ -68,6 +68,20 @@ model Host {
   @@index([scheduleId])
 }
 
+model CalVideoSettings {
+  eventTypeId Int       @id
+  eventType   EventType @relation(fields: [eventTypeId], references: [id], onDelete: Cascade)
+
+  disableRecordingForOrganizer Boolean @default(false)
+  disableRecordingForGuests    Boolean @default(false)
+  redirectUrlOnExit            String?
+
+  createdAt DateTime @default(now())
+  updatedAt DateTime @updatedAt
+
+  @@index([eventTypeId])
```

Comment:
This index isn't needed and will duplicate the index - it's already indexed by `@id`.

```suggestion
```

Reply:
Thanks. didn't knew this